docs/references/general/fitacf.md

```diff
@@ -1,2 +1,169 @@
+| *w_s_e* | *m/s* | *[num_pts]* | ***float*** | Spectral width error from sigma fit of ACF | 
+| *sd_l* | **None** | *[num_pts]* | ***float*** | Standard deviation of lambda fit | 
+| *sd_s* | **None** | *[num_pts]* | ***float*** | Standard deviation of sigma fit | 
+| *sd_phi* | **None** | *[num_pts]* | ***float*** | Standard deviation of phase fit of ACF | 
+| *x_qflg* | **None** | *[num_pts]* | ***char*** | Quality flag of XCF | 
+| *x_gflg* | **None** | *[num_pts]* | ***char*** | Ground scatter flag of XCF | 
+| *x_p_l* | *Db* | *[num_pts]* | ***float*** | Power from lamdba of XCF | 
+| *x_p_l_e* | *Db* | *[num_pts]* | ***float*** | Power error from lambda fit of XCF | 
+| *x_p_s* | *Db* | *[num_pts]* | ***float*** | Power from sigma fit of XCF | 
+| *x_p_s_e* | *Db* | *[num_pts]* |  ***float*** | Power  error from sigma fit  of XCF | 
+| *x_v* | *m/s* | *[num_pts]* | ***float*** | Velocity from XCF | 
+| *x_v_e* | *m/s* | *[num_pts]* | ***float*** | Velocity error from XCF | 
+| *x_w_l* | *m/s* | *[num_pts]* | ***float*** | Spectral width from lamdba fit of XCF | 
+| *x_w_l_e* | *m/s* | *[num_pts]* | ***float*** | Spectral width error from lamdbad fit of XCF | 
+| *x_w_s* | *m/s* | *[num_pts]* | ***float*** | Spectral wdith from sigma fit of XCF | 
+| *x_w_s_e* | *m/s* | *[num_pts]* | ***float*** | Spectral wdith error from sigma fit of XCF | 
+| *phi0* | **None** | *[num_pts]* | ***float*** | Phase determination at lag zero of the ACF | 
+| *phi0_e* | **None** | *[num_pts]* | ***float*** | Phase determination error at lag zero of the ACF | 
+| *elv* | *degrees* | *[num_pts]* | ***float*** | Elevation angle estimate |
+| *elv_low* | *degrees* | *[num_pts]* | ***float*** | Lowest estimated of elevation angle |
+| *elv_high* | *degrees* | *[num_pts]* | ***float*** | Highest estimate of elevation angle |
+| *x_sd_l* | **None** | *[num_pts]* | ***float*** | Standard deviation of lambda fit of XCF|
+| *x_sd_s* | **None** | *[num_pts]* | ***float*** | Standard deviation of sigma fit of XCF |
+| *x_sd_phi* | **None** | *[num_pts]* | ***float*** | Standard deviation of phase fit of XCF |
+
+## File structure
+
+FITACF files are comprised of records that record data nominally every 3 seconds for each integration time.
+Nominally each integration time will change the beam, `bmnum` indicates the beam number. However, some control programs like *camping mode* will stay on one beam for a whole scan. 
+A scan is determined with the `scan` flag by the value. 1 or -1 inidicates the beginning of scan then the value changes to 0 for the rest the scan. When the `scan` value changes from 0 to 1 this indicates the end of the scan. Nominally scans try to fit in a 60 second window. Older FITACF files may show scan times of 120 seconds. Certain *control programs* could change the scan length as well especially newer upcoming radars. During  *normal scan* beams will scan increasing/decreasing from 0
```

Comment:
```suggestion
A "scan" is a beam sequence which gets repeated. In the common mode (*normal scan*), one scan is completed when each beam has been sampled sequentially from 0 to `bmnum-1` (or `bmnum-1` to 0). Scans usually begin on whole-minute boundaries and last for either 1min or 2min. Custom control programs which define different scan lengths may also exist. The `scan` flag is used to indicate the beginning of each scan pattern. A value of 1 or -1 indicates the beginning of the scan, and then the value changes to 0 for the rest the scan. When the `scan` value changes from 0 back to 1 this indicates the end of the scan. 
```

I've removed references to "older/newer" radars & fitacf files since it's not really relevant to how a scan is defined.